pg_temp/pg_temp.py

```diff
@@ -358,6 +359,8 @@ def cleanup(self):
                 stdout=subprocess.DEVNULL,
             )
         elif self.pg_process:
+            pg_stop_cmd = ['pg_ctl', 'stop', '-D', self.pg_data_dir, '-m' 'fast']
+            self.run_cmd(pg_stop_cmd, level=2, bg=True).wait()
```

Comment:
I think since run_cmd with level 2 can potentially produce stdout and stderr, instead of `.wait()` we should call `.communicate()`. Also, since this command can be caught by the pg process or hang, we should have a timeout at which point we give up and call `kill()` anyway. Maybe 10 seconds?
The python docs have an example of how to do this properly, catching the `TimeoutExpired` exception https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate